### PR TITLE
Removed product query and receiving loading info from ProductPage

### PR DESCRIPTION
**Pull request by salesfelipe**
#### What is the purpose of this pull request?
Remove the product query from the ProductDetails component.

#### What problem is this solving?
Now the ProductDetail receives the product info instead of making the query itself  

#### How should this be manually tested?
[workspace](https://contentloader--storecomponents.myvtex.com/samsung-s9/p)

#### Types of changes

* [x] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/ProductDetails.js [hunk 1]
```diff
@@ -113,157 +109,145 @@ class ProductDetails extends Component {
   }
 
   render() {
-    const { slug, displayVertically } = this.props
+    const { product, loading, displayVertically } = this.props
```
Comment (claudivanfilho) [inline react/ProductDetails.js]: Seria interessante o product-details imitar o search-result. Nele o que é recebido nas props é o objeto `searchQuery`. 
https://github.com/vtex-apps/search-result/blob/d49ed558bd07b624a5721db28db3f4e1cafebca3/react/constants/propTypes.js#L141

**Review by claudivanfilho — CHANGES_REQUESTED** [review]
Faltou remover o arquivo `/graphql/productQuery.gql`

File: react/ProductDetails.js (hunk 1 above)
Comment (salesfelipe) [reply react/ProductDetails.js]: Why? I don´t receive a list of products!

Comment (salesfelipe) [reply react/ProductDetails.js]: 👍

**Review by andreldsa — APPROVED** [review]

**Review by claudivanfilho — APPROVED** [review]

**Pull request merged**